### ISSUE TITLE
[Question] Is X-Warp10-Token can be use in /api/v0/exec ?

### ISSUE THREAD

**Issue by ronronan** [senx/warp10-platform#1381]
When I do an HTTP request with option's method on `/api/v0/exec`, in headers, there is X-Warp10-Token` available, in `access-control-allow-headers`.

Is there any possible to use this header to inject Warp10's token in script when I call `/api/v0/exec` ?

**Comment by hbs** [comment]
Nope, tokens must be explicitely specified in the function calls  arguments which require them.

**Comment by ronronan** [comment]
Okay, I'll make a note of that. Thanks for your time.
